docs/tutorials/assets-tokens/withdrawing-tokens.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Withdraw tokens
+sidebar_position: 5
+hide_title: false
+vega_network: TESTNET
+description: Withdraw staked or claimed tokens
+---
+
+## Check the status of your tokens
+
+How to withdraw your VEGA tokens depends on the status of the tokens. You can withdraw any tokens that are *vested*, or that you already hold on your Ethereum address.
+
+In a nutshell, tokens may be locked for a time. Each epoch, a percentage of locked tokens begin vesting. Once they're vested, they are moved into a vested account. From there, you can redeem vested tokens by transferring them into your general account, at which point they can be withdrawn.
```

Comment:
This is vesting in Vega, I think we should be talking about Eth vesting here instead no?